samples/benchmarks/coremark/src/main.c

```diff
@@ -115,6 +117,19 @@ static int button_init(void)
 	return ret;
 }
 
+static void pre_run_action_perform(void)
+{
+	if (!IS_ENABLED(CONFIG_NRF_ETR)) {
+		return;
+	}
+
+	LOG_INF("Logging is stalled until the logging core finishes the CoreMark benchmark\n");
+
+	k_sleep(K_MSEC(100));
```

Comment:
How was this period chosen? experimentally? consider adding a define for it.

Reply:
Consider adding a comment why it's needed